java/gui/src/main/java/com/squareup/subzero/actions/SignTx.java

```diff
@@ -24,12 +25,23 @@
   public static CommandResponse.SignTxResponse signTx(SubzeroGui subzero,
       InternalCommandConnector conn, CommandRequest request,
       InternalCommandRequest.Builder internalRequest) throws IOException, NFException {
-
+    boolean isSerializedCommand = false;
```

Comment:
nit: simplify to `boolean isSerializedCommand = request.hasSerializedCommandRequest()`?